codice/alliance
# CAL-0729 Fixes NSILI and Video Admin Apps and removes unused webjars

#### What does this PR do?
Removes webjars that are exposed in DDF to resolve owasp finding and fixes admin apps.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@andrewkfiedler
@bdeining
@beyelerb
@brendan-hofmann
@brjeter
@clockard
@coyotesqrl
@figliold
@jaymcnallie
@jlcsmith
@kcwire
@lessarderic
@michaelmenousek
@millerw8
@mrmateo
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris
@stustison
@tbatie
@vinamartin
#### How should this be tested?
Build / Install / Install NSILI / Verify NSILI Admin Plugin works / Verify Video Admin Plugin works
#### Any background context you want to provide?
#### What are the relevant tickets?
https://github.com/codice/alliance/issues/729
#### Screenshots (if appropriate)
#### Checklist:
![Screen Shot 2019-07-10 at 4 12 17 PM](https://user-images.githubusercontent.com/6551038/61008699-8ca52600-a32d-11e9-8e96-c5ec36a24ce0.png)
![Screen Shot 2019-07-11 at 8 45 04 AM](https://user-images.githubusercontent.com/6551038/61060873-7d1eef00-a3b8-11e9-8b8f-cab32913bf42.png)
